.github/workflows/build-npm-release.yml

```diff
@@ -21,26 +18,28 @@ on:
   workflow_dispatch:
 
 jobs:
-  build-npm-release:
+  github-actions-ci:
     if : ${{ startsWith(github.ref, 'refs/tags/v') }}
     env:
-      YARN_TEST_OPTIONS: '--ci --coverage --colors'
+      YARN_TEST_OPTIONS: ''
       SQ_ROOT_DIR: './src'
-      PUBLISH_MOD_DESCRIPTOR: 'false'
-      COMPILE_TRANSLATION_FILES: 'false'
+      COMPILE_TRANSLATION_FILES: 'true'
+      PUBLISH_MOD_DESCRIPTOR: 'true'
       FOLIO_NPM_REGISTRY: 'https://repository.folio.org/repository/npm-folio/'
       FOLIO_NPM_REGISTRY_AUTH: '//repository.folio.org/repository/npm-folio/'
       FOLIO_MD_REGISTRY: 'https://folio-registry.dev.folio.org'
       NODEJS_VERSION: '18'
       JEST_JUNIT_OUTPUT_DIR: 'artifacts/jest-junit'
       JEST_COVERAGE_REPORT_DIR: 'artifacts/coverage-jest/lcov-report/'
+      BIGTEST_JUNIT_OUTPUT_DIR: 'artifacts/runTest'
+      BIGTEST_COVERAGE_REPORT_DIR: 'artifacts/coverage/lcov-report/'
       OKAPI_PULL: '{ "urls" : [ "https://folio-registry.dev.folio.org" ] }'
       SQ_LCOV_REPORT: 'artifacts/coverage-jest/lcov.info'
       SQ_EXCLUSIONS: '**/platform/alias-service.js,**/docs/**,**/node_modules/**,**/examples/**,**/artifacts/**,**/ci/**,Jenkinsfile,**/LICENSE,**/*.css,**/*.md,**/*.json,**/tests/**,**/stories/*.js,**/test/**,**/.stories.js,**/resources/bigtest/interactors/**,**/resources/bigtest/network/**,**/*-test.js,**/*.test.js,**/*-spec.js,**/karma.conf.js,**/jest.config.js'
 
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
+      - uses: folio-org/checkout@v2
```

Comment:
What does `folio-org/checkout` do?

Reply:
[nothing](https://folio-project.slack.com/archives/C58TABALV/p1675523785212679) :(

Reply:
Er. So why are we using it?

Reply:
@ncovercash -- As Mike, i do not understand why we continue to use a fork (or why using it in the first place) FOLIO-3698. Our fork has not been updated in over two years. That sounds like a dangerous glitch.

Reply:
Erk, sorry Noah that came out wrong. I sort of meant to say that we should get rid of any uses of that fork as soon as possible. Thanks for your efforts to attend to Workflows.